web/gridt_server/resources/announcement.py

```diff
@@ -0,0 +1,61 @@
+from flask import request
+from flask_restful import Resource
+from flask_jwt_extended import jwt_required, get_jwt_identity
+
+from gridt_server.schemas import AnnouncementSchema, UpdateAnnouncementSchema, DeleteAnnouncementSchema
+from gridtlib.exc import AnnouncementNotFoundError
+
+from .helpers import schema_loader
+
+from gridtlib.controllers.announcement import (
+    create_announcement,
+    _get_announcement,
```

Comment:
Remove unused import _get_announcement.

In python, we put _ or __ in front of function names when they are to be considered private.
Of course, python doesn't have private functions so strictly speaking, you are free to use them as another function. But this is considered to be bad practice because the developer intended the function not to be used.
Long story short please remove this import line.

Reply:
Done